### DISCUSSION TITLE
协议对应的代码

### DISCUSSION THREAD

**Discussion by clark-2468** [Mrs4s/go-cqhttp#2345] [Q&A] [answered]
在device.json里面的protocol的数字都对应什么设备a
我只知道3是MacOS
现在MacOS好像因为QQ限制了
我还没搞懂签名服务器
然后有人说换手表可以
所以我想问问都是几
谢谢

**Comment by dangyang1991** [comment, marked as answer]
Unset        ProtocolType = iota
	AndroidPhone             1
	AndroidWatch             2
	MacOS                    3
	QiDian                   4
	IPad                      5
	AndroidPad           6

**Reply by clark-2468** [reply]
多谢多谢
